proxy_load_balancer/src/domain/loadbalancer_strategy.rs

```diff
@@ -0,0 +1,5 @@
+
+
+pub trait LoadBalancingStrategy {
+    fn get_worker(&mut self, worker_hosts: Vec<String>) -> String;
```

Comment:
Hey Chris,
I'd suggest you to add the `new` method to the trait too.
Otherwise it could break your Application instantiating logic as it specifically depends on `new` constructor.